packages/components/pagination/src/page-navigator/page-navigator.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Value } from 'react-value';
+import PageNavigator from './page-navigator';
+
+const meta = {
+  title: 'Components/Pagination/PageNavigator',
+  component: PageNavigator,
+  tags: ['autodocs'],
+} satisfies Meta<typeof PageNavigator>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    totalPages: 10,
+    page: 1,
+  },
+  render: (args) => {
```

Comment:
Could we maybe use the helper here so we keep a compact code?

```ts
render: withControlledValue<TPageNavigatorProps>(PageNavigator)
```

Reply:
I pushed a refactor of the HOC to be able to use it here 7d8f97d48da932198d4a3145e874dd503250a9b1